meta-luneos/recipes-luneos/luneos-components/luneos-components.bb

```diff
@@ -5,7 +5,9 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
 PV = "0.4.2+git${SRCPV}"
 SRCREV = "6659065e96ac4a30ca08e8f75755c1c12009ec5a"
 
-DEPENDS = "qtbase qtdeclarative qtlocation qtquickcontrols qtquickcontrols2 luna-service2 libwebos-application kf5bluezqt-mer qtdeclarative-native"
+DEPENDS = "qtbase qtdeclarative qtlocation qtquickcontrols qtquickcontrols2 luna-service2 libwebos-application kf5bluezqt-mer qtdeclarative-native qtwebengine"
+
+require recipes-qt/qt5/qt5-git.inc
```

Comment:
This also sets SRC_URI, CVE_PRODUCT, PV which don't belong to luneos-components.bb. It would be better to store it in qtwebengine bbappappend together with chromium version.